analysis/basestats/parse_ssv.h

```diff
@@ -33,7 +33,7 @@ class SSVParser : public Parser {
                 break;
             }
             parsed=true;
-            batch.emplace_back(double(ts),oid,size);
+            batch.emplace_back(ts / (double)1e9,oid,size);
```

Comment:
Because ts is already a double, 1e9 does not need to be explicitly cast to double. It's also unusual to cast a static number (cleaner would be to write 1.0e9). --- minor change